docker/.env

```diff
@@ -1,8 +1,8 @@
 # component versions
 
-ARANGO_DB_VERSION=3.9.6
+ARANGO_DB_VERSION=3.10.3
 SPLINE_CORE_VERSION=0.7.7
-SPLINE_AGENT_VERSION=1.0.0
+SPLINE_AGENT_VERSION=1.0.4
```

Comment:
I wonder how is that diff possible? The current version of `.env` in the `main` branch contains `1.0.4` already.